bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/connection/Clip2Bridge.java

```diff
@@ -693,23 +797,31 @@ public Resources getResources(ResourceReference reference) throws ApiException,
      */
     private Resources getResourcesImpl(ResourceReference reference)
             throws HttpUnauthorizedException, ApiException, InterruptedException {
-        Session session = http2Session;
-        if (Objects.isNull(session) || session.isClosed()) {
-            throw new ApiException("HTTP 2 session is null or closed");
+        // work around for issue #15468
+        if (reference.getType() == ResourceType.ERROR) {
+            LOGGER.debug("Resource '{}' type unknown (issue #15468) => GET aborted", reference.getId());
```

Comment:
If we expect a quick firmware fix and don't want special treatment for `taurus_7455`, this seems more appropriate:
```suggestion
            // See issue #15468
            LOGGER.warning("Resource '{}' type unknown => GET aborted", reference.getId());
```

Otherwise we won't get warnings for other unknown resource types for which we are actually lacking support. Also, we should not mention this specific issue in the warning if we want to pretend it doesn't exist for the time being, or at least in context of this PR. Related to https://github.com/openhab/openhab-addons/pull/15477#discussion_r1304839672

Reply:
See my other comment just now.